python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -29,6 +29,18 @@
 
 logger = Ganga.Utility.logging.getLogger(modulename=1)
 
+_imported_GangaList = None
+
+do_not_copy = ['_proxyObject', '_proxyClass', '_data', '_index_cache', '_parent', '_registry', '_impl']
+
+def _getGangaList():
+    global _imported_GangaList
+    if _imported_GangaList is None:
+        from Ganga.GPIDev.Lib.GangaList.GangaList import GangaList
+        _imported_GangaList = GangaList
+    return _imported_GangaList
+
+
```

Comment:
given the class was here well before this commit I'm in favour of keeping it for now